doc/cephfs/add-remove-mds.rst

```diff
@@ -1,3 +1,10 @@
+.. note::
+   It is highly recommended to use `cephadm`_ or another Ceph orchestrator for
+   setting up the ceph cluster and thus the MDS(s). Use this approach only if
```

Comment:
s/ceph/Ceph/

Also suggest removing " and thus the MDS(s)", I think it's reasonable to subsume them as part of the cluster within a CephFS document.

Reply:
I've removed it from the note, thanks!